docs/source/metrics.rst

```diff
@@ -37,6 +37,13 @@ use the `output_transform` argument to transform it:
         metric = Accuracy(output_transform=output_transform)
         metric.attach(engine, "accuracy")
 
+Metrics could be combined together to form a new metric through arithmetics,
+for example:
+    .. code-block:: python
+        precision = Precision(average=False)
+        recall = Recall(average=False)
+        F1 = precision * recall * 2 / (precision + recall)
+
```

Comment:
Maybe we could add suplementary note point out that user should carefully think about combining metrics as F1 example and averaging as this can silently produce wrong result instead of expected.